app/bundles/CoreBundle/Entity/CommonRepository.php

```diff
@@ -1541,7 +1541,7 @@ protected function buildWhereClauseFromArray($query, array $clauses, $expr = nul
                                 $whereClause = $query->expr()->{$clause['expr']}($column, ':'.$param);
                                 $query->setParameter($param, $clause['val']);
                             }
-                            // no break
```

Comment:
The comment was added by CS Fixer (https://github.com/mautic/mautic/commit/ac66bcc8dac581d8b2ed8094f7f1e09ea7763515). So if a human thinks it should break then I believe it. Do you think it was intentionally missing?

Reply:
I do think it was intentional, but I'm only judging that based on the `// no break` comment

Reply:
Since the comment was added by CS Fixer and not a person and [the break was added by a person as a bug fix](https://github.com/mautic/mautic/commit/dcefd26c80cb44f6073a50d338b2d1424f055574), I'll leave it as is unless proven a bug.

Reply:
I see - thanks for looking into the history on it.